modules/cnv-example-bond-nncp.adoc

```diff
@@ -8,6 +8,14 @@
 Create a bond interface on nodes in the cluster by applying a `NodeNetworkConfigurationPolicy` manifest
 to the cluster. 
 
+[NOTE]
+====
+{CNVProductNameStart} 2.3 only supports the following bond modes: +
```

Comment:
The documentation is already versioned. Do we need to keep the version here? Seems like something that can get outdated quite fast.

Reply:
It'll be fixed before then though, right? :)